frontend/src/js/constants/Constants.js

```diff
@@ -107,15 +108,149 @@ const JOYRIDE_STEPS = [
             "Use this as an overlay or as an in-cell side-by-side comparison.",
             '</br><span style="color: red;">Clear Cell</span> will reset the cell back to the default. ',
             "This can be undone if you accidentally click it with the undo button.",
-            '</br><span style="color: blue;">Colormap Editor</span> will open a window for creating, editing, and applying colormaps.'
+            '</br><span style="color: blue;">Colormap Editor</span> will open a window for creating, editing, and applying colormaps.',
+            '</br><span style="color: purple;">Export</span> allows you to export/save the plot. NOT Implemented.',
+            '</br><span style="color: orange;">Calculator</span> allows you to create a new variable based on',
+            ' calculations performed on another variable.'
         ),
         selector: ".tools-container",
         position: "right",
         type: "click"
+    },
+    {
+        title: "Calculator",
+        text: "This button opens the calculator tool! Cool right?! Yeah it's cool :p",
+        selector: "#open-calculator-button",
+        position: "right",
+        type: "hover"
     }
 ];
 
+const CALCULATOR_STEPS = [
+    {
+        title: 'Calculator Tour',
+        text: 'This is a tour of the calculator and how to use it!',
+        selector: '#calculator-main',
+        position: 'left',
+        type: 'click'
+    },
+    {
+        title: 'Variables List',
+        text: "All the available variables that you've loaded are listed here.",
+        selector: '#calc-variable-region',
+        position: 'right',
+        type: 'click'
+    },
+    {
+        title: 'Derived Variable',
+        text: "Enter the name of the new variable you wish to derive here.",
+        selector: '#new-variable-name',
+        position: 'left',
+        type: 'click'
+    },
+    {
+        title: 'Calculation Area',
+        text: 'The calculation to be performed will be expressed here.'.concat(
```

Comment:
I think I would put this one before the previous one.
@sterlingbaldwin what do you think?

Reply:
You mean just re-ordering the help popups?

Reply:
yes, the order of the help. But now I'm not sure anymore, because then we would go back to it for regrid. And it might be weird. If we move regrid slide there too, then it might be too long before we get to the drag and drop section. So I guess it's ok as is.

Reply:
I think its fine for now. If we decide to change the order later it's pretty straightforward.